pkg/lang/javascript/aws_runtime/_fs.ts

```diff
@@ -123,7 +127,7 @@ async function s3_readFile(...args) {
 async function s3_readdir(path) {
     const bucketParams: ListObjectsCommandInput = {
         Bucket: bucketName,
-        Prefix: `${path}`,
+        Prefix: stripLeadingSlashes(`${path}`),
```

Comment:
Did this really used to be the behaviour? If so, we broke backwards compat, then we'll be breaking backwards compat again to return it.

Reply:
The actual behavior was actually quite a bit more complex as seen here:
https://github.com/hasnat/s3fs/blob/master/lib/utils.js

The key bit is that all the keys were normalized across host operating systems to match a specific format. Removing that from our FS breaks the expectation that you can use the Cloud FS the same as your local FS.